.github/workflows/rubyonrails.yml

```diff
@@ -0,0 +1,42 @@
+name: "Ruby on Rails CI"
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
This means only PRs that will be merged into `master` will be built. I've occasionally had to do a couple of PRs that patch the live server (we weren't ready to release `master` at that time). So as an extra safety net, I'd add `aws-eb-master` alongside `master` here.

Reply:
:+1: